django-server/server/api/urls.py

```diff
@@ -20,10 +20,16 @@
                                    CafeteriaFixedMenuOptionReviewViewSet,
                                    basename='cafeteria-fixed_menu_options-review')
 
+cameras_router = NestedSimpleRouter(router, r'cameras', lookup='camera')
+cameras_router.register(r'events', CameraEventsViewSet, basename='is_this_even_used?')
```

Comment:
keep the convention of naming the basename even it may not be used for now

Reply:
oh, I forgot about this 👍